seq/detail/window.cpp

```diff
@@ -109,9 +109,10 @@ bool Window::configExitGL()
 bool Window::processEvent( const eq::Event& event )
 {
     seq::Renderer* const renderer = getRenderer();
-    if( renderer->processEvent( event ))
-        return true;
-    return eq::Window::processEvent( event );
+    
+    bool successProcess = renderer->processEvent( event );
+
+    return eq::Window::processEvent( event ) || successProcess;
```

Comment:
I'm not sure about that. That potentially means that one event is handled twice. According to the documentation of Renderer::processEvent(), it should return false if the event is not handled by the renderer, hence it will be forwarded to eq::Window.